DEPENDENCIES.md

```diff
@@ -2788,8 +2823,8 @@ THE SOFTWARE.
 The following text applies to code linked from these dependencies:
 
 * [benchmark 0.1.0]( https://crates.io/crates/benchmark )
-* [glean-core 30.0.0]( https://github.com/mozilla/glean )
-* [glean-ffi 30.0.0]( https://github.com/mozilla/glean )
+* [glean-core 30.1.0]( https://github.com/mozilla/glean )
+* [glean-ffi 30.1.0]( https://github.com/mozilla/glean )
```

Comment:
I believe this applies to 31.0.0, since we're cutting this?

Reply:
This file is auto-generated (`bin/dependency-summary.sh`), so we just need to run this again against whatever is in master (and it now runs as part of `prepare-release.sh`, so gets updated in that)